starknet_py/net/signer/test_stark_curve_signer.py

```diff
@@ -57,3 +43,32 @@ def test_sign_transaction(transaction):
     assert len(signature) > 0
     assert all(isinstance(i, int) for i in signature)
     assert all(i != 0 for i in signature)
+
+
+# separate test due to contract_class using functions
```

Comment:
How does this affect the CI?

Reply:
After many changes in `read_contract` and using `pytest.skip()` inside it, **pytest** complained about the `read_contract` not being called from a test case.